README.md

```diff
@@ -1,7 +1,5 @@
-IBM DevOps training... 
 
-
-Release 1.0 on 11th March 2018
+Release 2.0 on 26th Aug 2018
```

Comment:
can you add date and time

Reply:
Hi, I have made the changes. Can you please review and approve?